microsoft-edge/devtools-guide-chromium/memory-inspector/memory-inspector-tool.md

```diff
@@ -21,48 +22,176 @@ ms.date: 02/15/2022
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.  -->
-# Inspect a JavaScript ArrayBuffer with the Memory Inspector tool
 
+# Inspect a JavaScript ArrayBuffer using Memory Inspector
+
+Use Memory Inspector to view JavaScript ArrayBuffers and the WebAssembly (WASM) resizable ArrayBuffer `WebAssembly.Memory`. You can open Memory Inspector from the [DevTools menu](#open-memory-inspector-from-the-devtools-menu) or [while debugging](#open-memory-inspector-while-debugging).
+
+<!-- add more scenario type info, ask Zoher -->
+
+
+<!-- ====================================================================== -->
+## Open the test site in Microsoft Edge
+
+1. Start Microsoft Edge.
+
+1. Open the test site [http://memory-inspector.glitch.me/demo-js.html](http://memory-inspector.glitch.me/demo-js.html). <!-- To do item: copy to Microsoft Repo and reference there -->
+
+
+<!-- ====================================================================== -->
+## Open Memory Inspector from the DevTools menu
```

Comment:
I'm not sure this section adds any value. The entry point for Memory Inspector really appears to be when you're debugging. I think we could cut this section and it wouldn't even affect the rest of the doc

Reply:
deleted